rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -1200,9 +1200,9 @@ Player::Player(const std::string & node_name, const rclcpp::NodeOptions & node_o
 : rclcpp::Node(node_name, node_options)
 {
   rosbag2_storage::StorageOptions storage_options =
-    get_storage_options_from_node_params(shared_from_this());
+    get_storage_options_from_node_params(this);
 
-  PlayOptions play_options = get_play_options_from_node_params(shared_from_this());
+  PlayOptions play_options = get_play_options_from_node_params(this);
```

Comment:
@roncapat Could you please clarify what was wrong with `shared_from_this` which you are replacing with raw pointer?